package.json

```diff
@@ -79,5 +79,8 @@
     "dns-over-http-resolver": "^2.1.1",
     "loupe": "^2.3.6",
     "testcontainers/**/nan": "^2.19.0"
+  },
+  "dependencies": {
+    "@lodestar/utils": "^1.29.0"
```

Comment:
why is this needed? this package is part of the work space, it shouldn't be required to install is globally